rebar.config

```diff
@@ -57,10 +57,10 @@
 {deps,
  [
   {lager, ".*", {git, "git://github.com/basho/lager", {branch, "1.2"}}},
-  {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", "v1.4.2"}},
+  {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", {tag, "v1.4.2"}}},
 
   %% for testing
-  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "1b773ee"}}, %"v1.1"}},
-  {meck, "0.8.2", {git, "git://github.com/eproxus/meck.git", "dde7590"}}
+  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {ref, "1b773ee"}}},
```

Comment:
I will look into your comments more deeply during the next week. This
weekend belongs to my son. And it's evenings my wife.

Philip Müller <notifications@github.com> schrieb am Fr., 19. Feb. 2016
17:09:

> In rebar.config
> <https://github.com/disco-framework/disco/pull/16#discussion_r53479839>:
>
> >
> >    %% for testing
> > -  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "1b773ee"}}, %"v1.1"}},
> > -  {meck, "0.8.2", {git, "git://github.com/eproxus/meck.git", "dde7590"}}
> > +  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {ref, "1b773ee"}}},
>
> (This comment should have been on your previous PR, but I noticed just now)
>
> Can we have a short comment explaining why we don't use the latest tagged
> version?
>
> It's a good rule to always explain version restrictions for dependencies.
> I've not followed the rule myself in this project before, but I think we
> should start with it sooner rather than later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/disco-framework/disco/pull/16/files#r53479839>.
>